standalone-metastore/packaging/src/assembly/src.xml

```diff
@@ -28,28 +28,38 @@
     <format>tar.gz</format>
   </formats>
 
-  <baseDirectory>apache-${project.artifactId}-${project.version}-src</baseDirectory>
+  <baseDirectory>apache-hive-metastore-${project.version}-src</baseDirectory>
 
   <fileSets>
     <fileSet>
-      <directory>${project.basedir}</directory>
+      <directory>${project.parent.basedir}</directory>
       <includes>
         <include>DEV-README</include>
```

Comment:
main packaging for some reason includes
````
  <include>.checkstyle</include>
  <include>.gitattributes</include>
  <include>.gitignore</include>
````

Reply:
these files are not present in the `standalone-metastore`, should I include them?